website/docs/components/card/partials/guidelines/guidelines.md

```diff
@@ -12,7 +12,23 @@ Static cards may have interactive elements within them but are not actionable or
 
 ### Interactive cards
 
-Interactive cards are actionable (e.g., it can be hovered, clicked, focused, etc.)
+Interactive cards are actionable (e.g., it can be hovered, clicked, focused, etc.). Ensure that if your cards link out to other places, do not have additional linkable elements within.
+
+!!! Dont
+
+Don't nest other interactive elements while the card itself is interactive. This is an accessibility violation called a nested interactive.
```

Comment:
```suggestion
Don't nest interactive elements within an interactive card. This is because screen reader users cannot access nested interactions properly. 
```